.oxlintrc.json5

```
@@ -0,0 +1,60 @@
+{
```

```suggestion
{
 	"$schema": "./node_modules/oxlint/configuration_schema.json",
```

https://oxc.rs/docs/guide/usage/linter/config-file-reference.html